formatTest/typeCheckedTests/expected_output/imperative.re

```diff
@@ -63,7 +63,7 @@ let shouldStillLoop = {contents: false};
 
 while (shouldStillLoop.contents) {
   print_string("You're in a while loop");
-  print_newline()
+  print_newline(())
```

Comment:
Same for this

Reply:
I removed all sugar regarding the last argument getting special printing, do we keep the last argument sugar when there is:
1) one argument
2) its type is unit

Reply:
Definitely! `foo(())` isn't great at all lol